flask_monitoringdashboard/database/function_calls.py

```diff
@@ -27,8 +27,10 @@ def add_function_call(time, endpoint, ip):
     """ Add a measurement to the database. """
     with session_scope() as db_session:
         group_by = None
-        if config.get_group_by:
+        try:
```

Comment:
i would still keep the conditional.
otherwise, there will always be an error, even for those 
cases where people don't install group_by